src/internals/utils.js

```diff
@@ -103,16 +118,41 @@ export function processTheme(theme) {
   return { definition, values };
 }
 
+const THEME_PRIORITIZED_KEYS = [
+  'colors',
+  'radii',
+  'sizes',
+  'space',
+  'borderStyles',
+  'borderWidths',
+  'fonts',
+  'fontSizes',
+  'fontWeights',
+  'lineHeights',
+  'zIndices',
+  'letterSpacings',
+];
+
+function searchHaveTokenConfigKey(theme, themeMap, key) {
```

Comment:
NIT: a better name for this function would be something like `getThemeScaleKey` and instead of using variable called `configKey` in various places in this file a more accurate name would be `scaleKey`.

Reply:
Okey, but colors key is also included. So, I'll rename getThemeKey and themeKey.